vllm/engine/llm_engine.py

```diff
@@ -474,6 +475,8 @@ def get_tokenizer_for_seq(sequence: Sequence) -> AnyTokenizer:
                 ),
             ))
 
+        self.group_id_to_holders: Dict[str, SeqGroupHolder] = {}
```

Comment:
Probably a better name?
```suggestion
        self.request_id_to_group_holders: Dict[str, SeqGroupHolder] = {}
```

Reply:
fixed in [245920f](https://github.com/vllm-project/vllm/pull/9302/commits/245920f17ae10675d98a4e5fc92a0e326cb3fc93)